test/e2e/framework.go

```diff
@@ -294,6 +302,9 @@ func (data *TestData) deployAntreaCommon(yamlFile string, extraOptions string) e
 
 // deployAntrea deploys Antrea with the standard manifest.
 func (data *TestData) deployAntrea() error {
+	if testOptions.enableCoverage {
```

Comment:
what about the IPsec testing? In that case we apply a different YAML (see below).

Reply:
Good catch, I updated hack/generate-manifest.sh and auto-generated antrea-coverage.yaml with ipsec